Horizon/Horizon/Sources/Features/AssignmentDetails/View/AssignmentDetailsViewModel.swift

```diff
@@ -147,7 +136,7 @@ final class AssignmentDetailsViewModel {
         aiEvents.sink {  [weak self] event, controller in
             switch event {
             case .assist:
-                self?.appEnvironment.router.route(to: "/tutor", from: controller)
+                self?.appEnvironment.router.show(AIAssembly.makeAITutorView(), from: controller, options: .modal(isDismissable: false))
```

Comment:
let's use `router.route(to:)` function and now that appEnvironment is not used anymore, we can delete the local variable and inject router directly to the vm

Reply:
Done